### PR TITLE
[SYCLomatic] Update migration desc of texLod APIs.

### PR DESCRIPTION
**Pull request by tangjj11**
Signed-off-by: Tang, Jiajun jiajun.tang@intel.com

### CODE REVIEW

File: clang/lib/DPCT/APINames.inc [hunk 1]
```diff
@@ -1432,13 +1432,13 @@ ENTRY(__syncthreads_and, __syncthreads_and, true, NO_FLAG, P0, "Successful: DPCT
 ENTRY(__syncthreads_or, __syncthreads_or, true, NO_FLAG, P0, "Successful: DPCT1078")
 ENTRY(tex1Dfetch, tex1Dfetch, true, NO_FLAG, P0, "Successful")
 ENTRY(tex1D, tex1D, true, NO_FLAG, P0, "Successful")
-ENTRY(tex1DLod, tex1DLod, true, NO_FLAG, P4, "DPCT1026/DPCT1027")
+ENTRY(tex1DLod, tex1DLod, true, NO_FLAG, P4, "DPCT1119")
```
Comment (daiyaan-ahmed6) [inline clang/lib/DPCT/APINames.inc]: `ENTRY(tex1DLod, tex1DLod, true, NO_FLAG, P4, "Successful/DPCT1119")`

tex1DLod can be migrated successfully or issue DPCT warning. I suggest we can use this as the comment.
Example: https://github.com/oneapi-src/SYCLomatic/blob/SYCLomatic/clang/lib/DPCT/TypeNames.inc#L138

File: clang/lib/DPCT/APINames.inc [hunk 2]
```diff
@@ -1432,13 +1432,13 @@ ENTRY(__syncthreads_and, __syncthreads_and, true, NO_FLAG, P0, "Successful: DPCT
...
 ENTRY(tex1DGrad, tex1DGrad, false, NO_FLAG, P4, "comment")
 ENTRY(tex2D, tex2D, true, NO_FLAG, P0, "Successful")
-ENTRY(tex2DLod, tex2DLod, true, NO_FLAG, P0, "DPCT1026/DPCT1027")
+ENTRY(tex2DLod, tex2DLod, true, NO_FLAG, P0, "DPCT1119")
```
Comment (daiyaan-ahmed6) [inline clang/lib/DPCT/APINames.inc]: Same as previous

File: clang/lib/DPCT/APINames.inc [hunk 3]
```diff
@@ -1432,13 +1432,13 @@ ENTRY(__syncthreads_and, __syncthreads_and, true, NO_FLAG, P0, "Successful: DPCT
...
 ENTRY(tex2DGrad, tex2DGrad, false, NO_FLAG, P0, "comment")
 ENTRY(tex3D, tex3D, true, NO_FLAG, P0, "Successful")
-ENTRY(tex3DLod, tex3DLod, true, NO_FLAG, P4, "DPCT1026/DPCT1027")
+ENTRY(tex3DLod, tex3DLod, true, NO_FLAG, P4, "DPCT1119")
```
Comment (daiyaan-ahmed6) [inline clang/lib/DPCT/APINames.inc]: Same as previous comment.

**Review by daiyaan-ahmed6 — COMMENTED** [review]
ENTRY(tex1DLod, tex1DLod, true, NO_FLAG, P4, "Successful/DPCT1119")

**Review by daiyaan-ahmed6 — APPROVED** [review]
LGTM!

**Review by ziranzha — APPROVED** [review]

**Review by zhimingwang36 — APPROVED** [review]

**Pull request merged**